### PR TITLE
Resprites The Detective, Blueshift and Constable Outfits

### PR DESCRIPTION
**Pull request by Imaginos16**
## About The Pull Request
Does what it says in the tin! This PR marks as one of the last remaining resprites of the department, with only a very small contingent of security-related clothing not touched as of currently.
This PR also renames the grey detective stuff to noir and/or dark, as those are more fitting to the new style:

image

image

This PR also removes the pantyhose alpha pixels from the female HoS parade outfit, since uniforms covering the entire legs with translucent pixels is a really, really bad practice.

## Why It's Good For The Game
We near a department being entirely redone, and updates old sprites into a newer standard!

Special thanks to @axietheaxolotl for making the jumpsuits, @triplezeta for creating the front frame of the detective trenchcoats, and @FuryMcFlurry for creating the blueshift armor and helmet!

## Changelog

:cl: Axietheaxolotl, TetraZeta, PositiveEntropy, FuryMcFlurry
imageadd: The detective outfits, the blueshift uniform and the constable outfits have been redone!
del: Removes the pantyhose from the HoS Parade Skirt
/:cl:

### CODE REVIEW

File: tools/UpdatePaths/clothingunderrepath.txt [hunk 1]
```diff
@@ -183,8 +183,8 @@
 /obj/item/clothing/under/rank/warden/navyblue : /obj/item/clothing/under/rank/security/warden/formal
 /obj/item/clothing/under/rank/det : /obj/item/clothing/under/rank/security/detective
 /obj/item/clothing/under/rank/det/skirt : /obj/item/clothing/under/rank/security/detective/skirt
-/obj/item/clothing/under/rank/det/grey : /obj/item/clothing/under/rank/security/detective/grey
-/obj/item/clothing/under/rank/det/grey/skirt : /obj/item/clothing/under/rank/security/detective/grey/skirt
+/obj/item/clothing/under/rank/det/grey : /obj/item/clothing/under/rank/security/detective/noir
+/obj/item/clothing/under/rank/det/grey/skirt : /obj/item/clothing/under/rank/security/detective/noir/skirt
```
Comment (Mothblocks) [inline tools/UpdatePaths/clothingunderrepath.txt:187]: Include update paths for `/obj/item/clothing/under/rank/security/detective/grey` and `/obj/item/clothing/under/rank/security/detective/grey/skirt` too.

Comment (Imaginos16) [reply tools/UpdatePaths/clothingunderrepath.txt:187]: Why, if you don't mind me asking? Not much of a coder, I was just changing paths.

**Comment by necromanceranne** [comment]
Might wanna double check the constable side to front sprites. It _looks_ like it gained a pixel in height on the sides.

**Comment by Imaginos16** [comment]
> Might wanna double check the constable side to front sprites. It _looks_ like it gained a pixel in height on the sides.

Hilariously, that's because of the bra my character has. Worry not, you can check the icondiffbot and it'll show its fine.

**Comment by Serxule** [comment]
fedora looks like a tophat, maybe have it shorter, but otherwise this is a banger

**Comment by Imaginos16** [comment]
> fedora looks like a tophat, maybe have it shorter, but otherwise this is a banger

Considered it, but it clips with hair, kind of like how it used to on the old version... Its why I made it a pixel taller.

**Review by Twaticus — APPROVED** [review]

**Comment by cacogen** [comment]
Does not look like a trenchcoat anymore.

**Comment by ComboTombo** [comment]
I have a few remarks regarding the detective’s new clothings, specifically the brown suit and coat. The older brown suit and trenchcoat was based on Lieutenant Columbo’s looks - despite the “suit” actually just being a shirt in the game - and achieved that by looking a bit worn out and old. The new clothing, however, seems entirely out of place. Not just with other clothes, mind you, but also with the code base itself. It looks like something that belongs to Goonstation’s style. I’d recommend using a beige/light brown palette, and also changing the trenchcoat texture to being a bit straighter on the sides, right now it looks like it’s tailored for someone with an hourglass body. Good work on it though.

**Comment by Mechanelize** [comment]
the detective coat sprite is terrible

**Pull request merged**